pom.xml

```diff
@@ -4,16 +4,16 @@
 
     <groupId>com.currencycloud.currencycloud-java</groupId>
     <artifactId>currencycloud-java</artifactId>
-    <version>0.7.9-SNAPSHOT</version>
+    <version>0.9.1</version>
```

Comment:
This should be -SNAPSHOT - the mvn release lifecycle is responsible for promotion itself, not the pom.

Reply:
Yep. As per my comment above. See: https://github.com/CurrencyCloud/currencycloud-java/wiki/Releasing-the-SDK

Reply:
Changed to conform with contributing guidelines